src/java/com/puppetlabs/ssl_utils/ExtensionsUtils.java

```diff
@@ -864,7 +891,17 @@ private static Integer getGnTagFromName(String name) {
                 if (ret.get(type) == null) {
                     ret.put(type, new ArrayList<String>());
                 }
-                String name = (String) asn1ObjToObj(generalName.getName());
+
+                String name;
+                switch (generalName.getTagNo()) {
+                    case GeneralName.iPAddress:
+                        name = octetStringToIpString((ASN1OctetString)generalName.getName());
```

Comment:
Just curious - what do you get if you just do what would happen in the default case here instead - a `.toString` instead of calling `octetStringToIpString`.  Something much less useful, presumably?